include/rego/passes.h

```diff
@@ -31,10 +31,12 @@ namespace rego
   PassDef explicit_enums();
   PassDef body_locals(const BuiltIns& builtins);
   PassDef value_locals(const BuiltIns& builtins);
-  PassDef compr();
+  PassDef compr1();
```

Comment:
I've been trying to keep the names vaguely related to what happens in the past in most cases. Perhaps you can change this to `rules_to_compr`?